src/undate/converters/calendars/hebrew/converter.py

```diff
@@ -0,0 +1,71 @@
+from typing import Union
+
+from convertdate import hebrew  # type: ignore
+from lark.exceptions import UnexpectedCharacters
+
+from undate.converters.base import BaseCalendarConverter
+from undate.converters.calendars.hebrew.parser import hebrew_parser
+from undate.converters.calendars.hebrew.transformer import HebrewDateTransformer
+from undate.undate import Undate, UndateInterval
+
+
+class HebrewDateConverter(BaseCalendarConverter):
+    """
+    Converter for Hebrew Anno Mundicalendar.
+
+    Support for parsing Anno Mundi dates and converting to Undate and UndateInterval
+    objects in the Gregorian calendar.
+    """
+
+    #: converter name: Hebrew
+    name: str = "Hebrew"
+    calendar_name: str = "Anno Mundi"
+
+    def __init__(self):
+        self.transformer = HebrewDateTransformer()
+
+    def min_month(self) -> int:
+        """first numeric month for the specified year in this calendar"""
+        # hebrew calendar civil year starts in Tishri
+        return hebrew.TISHRI
+
+    def max_month(self) -> int:
+        """last numeric month for the specified year in this calendar"""
+        # hebrew calendar civil year starts in Tishri
+        # Elul is the month before Tishri
+        return hebrew.ELUL
+
+    def max_day(self, year: int, month: int) -> int:
+        """maximum numeric day for the specified year and month in this calendar"""
+        # NOTE: unreleased v2.4.1 of convertdate standardizes month_days to month_length
```

Comment:
Is there a maintainer for `convertdate`? Looks like that branch has been hanging for a couple of years. Not sure if that's an issue if the library is doing what we need it to

Reply:
You're right, it does look pretty unmaintained. My PR a couple of years ago was merged in pretty quickly but never released. I see some open PRs hanging out that look kind of old...

Not great, but the existing calendar conversion logic is enough for what we need now, so I'm inclined to keep using it until it becomes a problem. I'm not aware of another comparable option in python.